src/app/parcels/batch/BatchParcelsReducer.test.ts

```diff
@@ -329,65 +329,67 @@ describe("reducer", () => {
         expect(noColumnOverridenState.clientOverrides).toEqual([]);
     });
 
-    it("should autofill for an existing client and format result correctly", async () => {
-        const actionAddRow: BatchActionType = {
-            type: "add_row",
-        };
+    // When the use existing client method is implemented in the reducer use this test
 
-        const lastRowEmptyState1 = await batchParcelsReducer(mockTableDataState, actionAddRow);
+    // it("should autofill for an existing client and format result correctly", async () => {
+    //     const actionAddRow: BatchActionType = {
+    //         type: "add_row",
+    //     };
```

Comment:
Rather than commenting out the test, can we [mark it as skipped ](https://jestjs.io/docs/api#testskipname-fn). That way it'll be listed in the test reports as a skipped test so we won't lose track of it accidentally